src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect } from "react";
+import { fetchArticles, fetchArticlesByTopic } from "../api";
+import { useParams, Link } from "react-router-dom";
+
+export default function ArticlesList() {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  //   const { topic } = useParams();
```

Comment:
Commented-out code shouldn't be included in the PR. Make sure to delete this before merging and add it to the branch for the next ticket.